test/it/uk/gov/hmrc/individualsifapistub/DetailsRepositorySpec.scala

```diff
@@ -73,10 +72,11 @@ class DetailsRepositorySpec extends RepositoryTestHelper with TestHelpers {
       val id = s"${ident.nino.getOrElse(ident.trn.get)}-$useCase"
 
       val detailsResponse = DetailsResponse(id, request.contactDetails, request.residences)
+      val returnVal       = DetailsResponseNoId(detailsResponse.contactDetails, detailsResponse.residences)
```

Comment:
Same as above

Reply:
@parkeradam  - again we need to do this to drop the id from the response.